src/Traits/SoftDeletes.js

```diff
@@ -123,6 +128,12 @@ class SoftDeletes {
         }
       },
 
+      columnName: {
```

Comment:
I don't like that this property is called `columnName`. IMO it should be something more descriptive (i.e., `deletedColumnName`).